src/mon/OSDMonitor.cc

```diff
@@ -11316,6 +11319,22 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
 	}
       }
 
+      string technique = profile_map["technique"];
```

Comment:
I'd rather allow tested and known-good ec profiles than disallow specifically blaum_roth.

Reply:
jerasure/reed_sol_van I think is the default.  @neha-ojha @rzarzynski do we have any other recommended configurations?

Reply:
So something like:

```
if (technique != reed_sol_van) {
   // issue experimental ec profile warning
}
```

In addition to checking against any other approved technique?

Reply:
It should probably be a static method in src/erasure/code/ErasureCode.h, perhaps ErasureCode::is_experimental_profile which takes the whole profile_map.  Otherwise, yeah, that's the idea (along with any other we deem appropriate for users to use without extra information).

Reply:
Added.